server/api/router.go

```diff
@@ -64,6 +64,7 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	storeHandler := newStoreHandler(svr, rd)
 	router.HandleFunc("/api/v1/store/{id}", storeHandler.Get).Methods("GET")
 	router.HandleFunc("/api/v1/store/{id}", storeHandler.Delete).Methods("DELETE")
+	router.HandleFunc("/api/v1/store/remove-tombstone", storeHandler.RemoveTombStone).Methods("DELETE")
```

Comment:
I think it's better to put it into `stores` instead of `store`?

Reply:
Fine, I'll move it to `storesHandler` tonight. :ok_hand: